packages/buttons/src/styled/StyledButton.ts

```diff
@@ -224,7 +222,8 @@ const sizeStyles = (props: IStyledButtonProps & ThemeProps<DefaultTheme>) => {
 
     retVal = css`
       padding: 0 ${em(math(`${padding} - ${props.theme.borderWidths.sm}`), fontSize)};
-      line-height: ${math(`${lineHeight} - (${props.theme.borderWidths.sm} * 2)`)};
+      height: ${height};
+      line-height: ${lineHeight};
```

Comment:
Out of curiosity, since button text doesn't wrap in Garden (or do they?), what are the consequences of allowing the browser to handle line-height for buttons? Alternatively, what are the consequences of setting the `line-height` value to `1`?

I tried this out in the `basic.md` example, and it (visually) looks the same.

Reply:
This could be legacy. As I remember, the use of `line-height` ensures a consistent result across elements (i.e. `a` or `input type="button"`), browsers, and fonts (we used to use custom).